pypeit/pypeit.py

```diff
@@ -804,7 +804,7 @@ def objfind_one(self, frames, det, bg_frames, std_outfile=None):
                             slits=True, clear=True)
 
         # Do it
-        global_sky, sobjs_obj, skymask = self.redux.run_objfind(std_trace=std_trace, show_peaks=self.show)
+        global_sky, sobjs_obj, skymask = self.redux.run_objfind(std_trace=std_trace)
```

Comment:
Why are you disabling self.show? I think we still want that option, which as per my comment above can be achieved if we specify whether we want interactive QA, or the QA plot to disk.